tests/outputs_test.go

```diff
@@ -0,0 +1,76 @@
+package tests
```

Comment:
Did this test need `// +build integration` at the top?  Or is it runnin' right w/o it?